tests/rptest/services/redpanda_cloud.py

```diff
@@ -101,39 +134,40 @@ def partitions_upper_limit(self):
     #    |        |        |   |         |          |   +- partitions_max
     #    |        |        |   |         |          |   |      +- connections_limit
     #    |        |        |   |         |          |   |      |     +- memory_per_broker
-    #    |        |        |   |         |          |   |      |     |
+    #    |        |        |   |         |          |   |      |     |       +- machine_type
+    #    |        |        |   |         |          |   |      |     |       |
     CloudTierName.AWS_1: AdvertisedTierConfig(
-         20*MiB,  60*MiB,  3,  512*MiB,  300*GiB,   20, 1000,  1500, 32*GiB
+         20*MiB,  60*MiB,  3,  512*MiB,  300*GiB,   20, 1000,  1500, 32*GiB, MachineTypeName.I3EN_XLARGE,
     ),
     CloudTierName.AWS_2: AdvertisedTierConfig(
-         50*MiB, 150*MiB,  3,  512*MiB,  500*GiB,   50, 2000,  3750, 64*GiB
+         50*MiB, 150*MiB,  3,  512*MiB,  500*GiB,   50, 2000,  3750, 64*GiB, MachineTypeName.I3EN_2XLARGE,
```

Comment:
Seems like the `memory_per_broker` should really be moved into the machine type stuff since otherwise there is unnecessary possibility of the machine type & memory value being inconsistent, but it could be in a later change.